contracts/hackatom/src/contract.rs

```diff
@@ -26,7 +26,7 @@ pub struct InitMsg {
 /// by blockchain logic (in the future by blockchain governance)
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct MigrateMsg {
-    pub verifier: HumanAddr,
+    pub payout: HumanAddr,
```

Comment:
Every contract has its own `MigrateMsg`. The fields can be different, depending on the migration logic. In the `burner` contract we have a `payout` address. In the `hackatom` contract, we need a `verifier` address.

Could you change it back here (hackatom only)?